cl/search/templates/advanced.html

```diff
@@ -659,7 +659,7 @@ <h3>Background</h3>
                 <p>If you find this work valuable, or if it saves you money on your PACER bill, please donate.
                 </p>
                 <p>
-                    <a href="{% url "donate" %}?referrer=recap-adv-search"
+                    <a href="https://donate.free.law/forms/supportflp"
```

Comment:
Same here. "Please donate or become a member," and then two buttons?

Reply:
Sure!